### ISSUE TITLE
how to change global state not based on onIndexChange which is slow

### ISSUE THREAD

**Issue by yonatanmn** [react-native-community/react-native-tab-view#463]
onIndexChange is called only after animation is finished and some time passes. I guess this is intended.

I'm looking for a way to update other component (in other place of the react tree) with the requested index. Something similar to TabBar changing view **with** swiping, just somewhere else.
For that i'll need the props passed to TabBar passed in events - `onPositionChange` for example. 

current behaviour looks slow\buggy - only after swipe ended the other component updated.

thanks

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
